baselines/Common Controls Minimum Viable Secure Configuration Baseline v0.2.md

```diff
@@ -1139,3 +1130,38 @@ To configure Supplemental Data Storage per the policy:
 3.	Click the **Supplemental Data Storage** card.
 4.	Ensure the checkbox for "**Russian Federation**" is unchecked.
 6.	Click **Save**.
+
+
+## 16. Additional Google Services
+
+This section covers the Google services that do not have an individual control and whether these services are on or off.
+
+### Policy
+
+#### GWS.COMMONCONTROLS.16.1v0.1
+Service status SHOULD be set to OFF for everyone.
+
+- _Rationale_: Allowing access to additional google services without a need creates unnceccessary vulnerabilities within the Google Workspace enviroment. By turning these services off, it mitigates the risk by not allowing access.
```

Comment:
- "creates vulnerabilities" is too confident. It seems to imply that other Google services are vulnerable, which may be a possibility, but is far from certain. "May create" might be better.
- Two spelling errors
```suggestion
- _Rationale_: Allowing access to additional google services without a need may create unnecessary vulnerabilities within the Google Workspace environment. By turning these services off, it mitigates the risk by not allowing access.
```